### PR TITLE
Accept null for many builder entries

### PR DESCRIPTION
**Pull request by mbax**

### CODE REVIEW

File: src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java [hunk 1]
```diff
@@ -65,7 +66,7 @@ public AwayCommand message(@Nonnull String message) {
      * @return this command
      */
     @Nonnull
-    public AwayCommand messageRemove() {
+    public AwayCommand notAway() {
```
Comment (lol768) [inline src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java]: Not sure about this name. Maybe `unaway`? I don't know

**Comment by lol768** [comment]
LGTM, any uhh documentation changes that need to be made following this being merged in or do most examples not make use of the removed methods anyway?

**Comment by mbax** [comment]
I don't see any docs referencing it. :)

File: src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java (hunk 1 above)
Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java]: `unaway` would match RFC 1459's name for the numeric - RPL_UNAWAY for that state. The description however (5.1), describes it as "to remove the AWAY message" so perhaps `messageRemove` *was* the best answer?

Comment (lol768) [inline src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java]: Yeah, I'd be pretty happy if it was left as `messageRemove` - I don't see it as the same as the ones on the `ClientBuilder` which you've removed.

**Comment by lol768** [comment]
Great!

File: src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java (hunk 1 above)
Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java]: Alrighty!

**Comment by lol768** [comment]
LGTM to merge now :)

File: src/test/java/org/kitteh/irc/client/library/command/AwayCommandTest.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ public void testNoNewAnymore() {
 
         AwayCommand awayCommand = new AwayCommand(client);
         awayCommand.message(MESSAGE);
-        awayCommand.messageRemove();
+        awayCommand.notAway();
```
Comment (lol768) [inline src/test/java/org/kitteh/irc/client/library/command/AwayCommandTest.java]: Needs updating I think @mbax

**Comment by mbax** [comment]
https://github.com/KittehOrg/KittehIRCClientLib/commit/3d5a322d823a46d5b800509e58f7370f24cbdc11

**Pull request closed without merging**